src/components/secret-display/SecretDisplaySection.tsx

```diff
@@ -13,98 +13,49 @@ import Spinner from '../Spinner';
 import { useWallets } from '@rainbow-me/hooks';
 import styled from '@rainbow-me/styled-components';
-import { margin, padding, position, shadow } from '@rainbow-me/styles';
+import { margin, position, shadow } from '@rainbow-me/styles';
+import { useTheme } from '@rainbow-me/theme';
 import logger from 'logger';
 
-const Title = styled(Text).attrs({
-  align: 'center',
-  size: 'lmedium',
-  weight: 'bold',
-})({
-  paddingTop: isSmallPhone => (isSmallPhone ? 0 : 20),
-});
-
-const DescriptionText = styled(Text).attrs(({ theme: { colors } }) => ({
-  align: 'center',
-  color: colors.alpha(colors.blueGreyDark, 0.6),
-  lineHeight: 'loose',
-  size: 'lmedium',
-  weight: 'semibold',
-}))({
-  marginBottom: 42,
-  marginTop: 5,
-  paddingHorizontal: 3,
-});
-
-const AuthenticationText = styled(Text).attrs({
-  align: 'center',
-  color: 'blueGreyDark',
-  size: 'large',
-  weight: 'normal',
-})({
-  ...padding.object(0, 60),
-});
-
-const CopyButtonIcon = styled(Icon).attrs(({ theme: { colors } }) => ({
+const CopyButtonIcon = styled(Icon).attrs(({ theme: { colors } }: any) => ({
   color: colors.appleBlue,
   name: 'copy',
 }))({
   ...position.sizeAsObject(16),
   marginTop: 0.5,
 });
 
-const CopyButtonRow = styled(RowWithMargins).attrs({
-  align: 'center',
-  justify: 'start',
-  margin: 6,
-})({
-  backgroundColor: ({ theme: { colors } }) => colors.transparent,
-  height: 34,
-});
-
-const CopyButtonText = styled(Text).attrs(({ theme: { colors } }) => ({
-  color: colors.appleBlue,
-  letterSpacing: 'roundedMedium',
-  lineHeight: 19,
-  size: 'lmedium',
-  weight: 'bold',
-}))({});
-
-const ToggleSecretButton = styled(Button)(({ theme: { colors } }) => ({
+const ToggleSecretButton = styled(Button)(({ theme: { colors } }: any) => ({
   ...margin.object(0, 20),
   ...shadow.buildAsObject(0, 5, 15, colors.purple, 0.3),
   backgroundColor: colors.appleBlue,
 }));
 
-const BiometryWarningText = styled(Text).attrs(({ theme: { colors } }) => ({
-  align: 'center',
-  color: colors.alpha(colors.blueGreyDark, 0.6),
-  lineHeight: 'looser',
-  size: 'lmedium',
-}))({
-  ...padding.object(0, 35),
-});
-
 const LoadingSpinner = android ? Spinner : ActivityIndicator;
 
+interface SecretDisplaySectionProps {
+  onSecretLoaded?: (seedExists: boolean) => void;
+  onWalletTypeIdentified?: (walletType: EthereumWalletType) => void;
+}
+
 export default function SecretDisplaySection({
-  isSmallPhone,
   onSecretLoaded,
   onWalletTypeIdentified,
-}) {
+}: SecretDisplaySectionProps) {
   const { params } = useRoute();
   const { selectedWallet, wallets } = useWallets();
-  const walletId = params?.walletId || selectedWallet.id;
+  const walletId = (params as any)?.walletId || selectedWallet.id;
   const currentWallet = wallets[walletId];
   const [visible, setVisible] = useState(true);
   const [isRecoveryPhraseVisible, setIsRecoveryPhraseVisible] = useState(false);
-  const [seed, setSeed] = useState(null);
+  const [seed, setSeed] = useState(null as any);
```

Comment:
is the `as any` needed? maybe you want `useState<string | null>(null)` ?

Reply:
i just wanted the ts warning to go away. i didn't know about this way to do it which looks way better